### PR TITLE
Switch the identitifer to a new unique id for new connections

### PR DESCRIPTION
**Pull request by manodnyab**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Description

Switch each connection identifier from start_url + region combination to a random UIUD to be able to distinguish between connections having different start scopes but same start url and region.

 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: plugins/core/core/src/software/aws/toolkits/core/credentials/ToolkitCredentialsProvider.kt [hunk 1]
```diff
@@ -128,7 +129,7 @@ interface ToolkitBearerTokenProviderDelegate : SdkTokenProvider, ToolkitAuthenti
 class ToolkitBearerTokenProvider(val delegate: ToolkitBearerTokenProviderDelegate) : SdkTokenProvider by delegate, ToolkitAuthenticationProvider by delegate {
     companion object {
         // TODO: is there a better place for this
-        fun ssoIdentifier(startUrl: String, region: String = DEFAULT_SSO_REGION) = "sso;$region;$startUrl"
+        fun ssoIdentifier() =  UUID.randomUUID().toString()
```
Comment (rli) [inline plugins/core/core/src/software/aws/toolkits/core/credentials/ToolkitCredentialsProvider.kt:132]: doesn't this need to be stable? i think connection pinning is keyed off of this value

File: plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/credentials/DefaultToolkitAuthManager.kt [hunk 2]
```diff
@@ -143,25 +143,7 @@ class DefaultToolkitAuthManager : ToolkitAuthManager, PersistentStateComponent<T
...
-        transientConnections.removeAll { existOldConn ->
-            (existOldConn.id == newConnection.id).also { isDuplicate ->
-                if (isDuplicate && existOldConn is Disposable) {
-                    ApplicationManager.getApplication().messageBus.syncPublisher(BearerTokenProviderListener.TOPIC)
-                        .onChange(existOldConn.id, newConnection.scopes)
-                    Disposer.dispose(existOldConn)
-                }
-            }
-        }
+// TODO: handle the case for existing connections
```
Comment (rli) [inline plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/credentials/DefaultToolkitAuthManager.kt:146]: ?

File: plugins/core/webview/src/q-ui/components/toolkitOptions.vue [hunk 3]
```diff
@@ -82,9 +82,9 @@ export default defineComponent({
         feature(): Feature {
             return this.$store.state.feature
         },
-        existConnections(): AwsBearerTokenConnection[] {
-            return this.$store.state.existingConnections
-        }
+        // existConnections(): AwsBearerTokenConnection[] {
+        //     return this.$store.state.existingConnections
+        // }
```
Comment (rli) [inline plugins/core/webview/src/q-ui/components/toolkitOptions.vue:87]: don't we still need this?

Comment (manodnyab) [reply plugins/core/webview/src/q-ui/components/toolkitOptions.vue:87]: Yes, we do, the next commit will revert this

**Comment by manodnyab** [comment]
This PR is no longer required

**Pull request closed without merging**